sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -346,20 +348,28 @@ def test_explicit_set_string_index_type(self, client):
         poller.result()
 
     @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer(client_kwargs={"api_version": TextAnalyticsApiVersion.V3_1_PREVIEW_3})
+    @TextAnalyticsClientPreparer()
     def test_bidirectional_relation_type(self, client):
         result = list(client.begin_analyze_healthcare_entities(
             documents=["The patient was diagnosed with Parkinsons Disease (PD)"]
         ).result())
 
-        self.assertEqual(len(result[0].entities), 2)
-        entity1 = list(filter(lambda x: x.text == "Parkinsons Disease", result[0].entities))[0]
-        entity2 = list(filter(lambda x: x.text == "PD", result[0].entities))[0]
+        assert len(result) == 1
```

Comment:
is the `len()` for the pages?

Reply:
yeah, this is asserting we only have one main response object (one page) returned